rules/aep0004/resource_plural.go

```diff
@@ -31,7 +31,7 @@ var resourcePlural = &lint.MessageRule{
 		r := utils.GetResource(m)
 		l := locations.MessageResource(m)
 		p := r.GetPlural()
-		pLower := utils.ToLowerCamelCase(p)
+		pLower := utils.ToKebobCase(p)
```

Comment:
```suggestion
		pLower := utils.ToKebabCase(p)
```

Here and elsewhere :)